File: exapmles.proj/ui/uilabel/uilabelwithuifont/uilabelwithuifont/ViewController.m
//
//  ViewController.m
//  uilabelwithuifont
//
//  Created by CZ on 3/12/16.
//  Copyright © 2016 projm. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    
    NSArray *fonts = [UIFont familyNames];
    for ( NSString *font in fonts) {
        NSLog(@"Font Family:%@ and Font Name %@", font, [UIFont fontNamesForFamilyName:font]);
    }
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: blog/iOS/ui/uifont/example/UIFontDemo/UIFontDemo/ViewController.swift
//
//  ViewController.swift
//  UIFontDemo
//
//  Created by apollo on 17/11/2016.
//  Copyright © 2016 projm. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    func layoutUI() {
        let font = UIFont.systemFont(ofSize: 12.0)
        let pfont = UIFont.preferredFont(forTextStyle: .body)
    }
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        self.layoutUI()
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

